charts/kubewarden-crds/chart-values.yaml

```diff
@@ -0,0 +1,2 @@
+policyReports:
+  enable: true
```

Comment:
Maybe we should add an inline comment, suggesting the user to set this to false if the Policy Report CRDs are already installed on the cluster

Reply:
done in https://github.com/kubewarden/helm-charts/pull/239/commits/b08e5badb5cd3458119e0db79ac9f1253b21361f.